assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -549,13 +551,20 @@ const MapDisplay = ({
       shapes={[]}
       stateClasses={stateClasses}
     >
-      <SelectionDataLayers
-        selectedEntity={selectedEntity}
-        showSelectionCard={showSelectionCard}
-        deleteSelection={deleteSelection}
-        setSelection={setSelection}
-        setStateClasses={setStateClasses}
-      />
+      <AutoPanPaddingConfig.Provider
```

Comment:
question (non-blocking): Does this need to wrap `<SelectionDataLayers/>`, or only the relevant `<RouteStopMarkers/>` element? I lean towards having it wrap at the lowest level it is needed and pulling it up to a higher level as needed, so it is easier to see how it is used, but that is just my preference!

Reply:
That makes sense to me!